styles/storybook.scss

```diff
@@ -0,0 +1,21 @@
+/* Storybook requires different paths for variables,
+which requires a custom import order to replace those settings */
+@import '@trussworks/react-uswds/lib/index.css';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-general';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-typography';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-spacing';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-color';
+@import 'node_modules/uswds/dist/scss/theme/uswds-theme-utilities';
+
+$theme-show-notifications: false;
+/* GENERAL */
+$theme-image-path: '../node_modules/uswds/dist/img';
+
+/* TYPOGRAPHY */
+$theme-font-path: '../node_modules/uswds/dist/fonts';
+
+/* COMPONENTS */
+$theme-hero-image: '../node_modules/uswds/dist/img/hero.png';
+
+@import 'node_modules/uswds/src/stylesheets/packages/_required.scss';
+@import 'node_modules/uswds/dist/scss/uswds.scss';
```

Comment:
NOT a blocker: I was curious so I tried replacing all of the `node_modules` references in this file with `~` and what do you know, Storybook imports external CSS properly!! so this might be a little bit cleaner if you feel like changing it (and then maybe if `~` starts working in NextJS in the future we won't need a separate file for Storybook)

```
/* Storybook requires different paths for variables,
which requires a custom import order to replace those settings */
@import '@trussworks/react-uswds/lib/index.css';
@import '~uswds/dist/scss/theme/uswds-theme-general';
@import '~uswds/dist/scss/theme/uswds-theme-typography';
@import '~uswds/dist/scss/theme/uswds-theme-spacing';
@import '~uswds/dist/scss/theme/uswds-theme-color';
@import '~uswds/dist/scss/theme/uswds-theme-utilities';

$theme-show-notifications: false;
/* GENERAL */
$theme-image-path: '~uswds/dist/img';

/* TYPOGRAPHY */
$theme-font-path: '~uswds/dist/fonts';

/* COMPONENTS */
$theme-hero-image: '~uswds/dist/img/hero.png';

@import '~uswds/src/stylesheets/packages/_required.scss';
@import '~uswds/dist/scss/uswds.scss';
```

Reply:
done 🎉